frontend/style/filterpage.css

```diff
@@ -0,0 +1,18 @@
+.FilterPage {
```

Comment:
no first-letter capitalize for css classes. Rename it in filterPage. 
Remember to change in jsx too.